OpenRA.Mods.Common/Traits/Power/CanPowerDown.cs

```diff
@@ -34,18 +34,43 @@ public class CanPowerDownInfo : ConditionalTraitInfo, Requires<PowerInfo>
 		public override object Create(ActorInitializer init) { return new CanPowerDown(init.Self, this); }
 	}
 
-	public class CanPowerDown : ConditionalTrait<CanPowerDownInfo>, IPowerModifier, IResolveOrder, IDisable, INotifyOwnerChanged
+	public class CanPowerDown : ConditionalTrait<CanPowerDownInfo>, IPowerModifier, IResolveOrder, INotifyOwnerChanged
 	{
-		[Sync] bool disabled = false;
+		bool disabled = false;
```

Comment:
We probably want to keep this `[Sync]`

Reply:
Can we please rename this to `poweredDown` to avoid confusion with `IsTraitDisabled`?